OpenShift-Installation/Openshift-Installation.rst

```diff
@@ -193,11 +161,11 @@ Setup
 Installation for a Single Master
 -----------------------------------
 
-1. Create a nodes file for Ansible configuration for a single master in the openshift-ansible directory with the contents shown below.
+1. Create a nodes or inventory file for Ansible configuration for a single master in the openshift-ansible directory with the contents shown below.
 
 2. Verify that the image versions are accurate by checking the TAG displayed by 'docker images' output for successful deployment of Nuage daemonsets: 
 
-  .. Note:: The following nodes file is provided as a sample. Please update the values with your actual deployment. The below nodes file deploys OpenShift version 3.7.9. To deploy OpenShift version 3.6, use 'openshift_pkg_version=-3.6.173.0.5' or to deploy OpenShift version 3.5, use 'openshift_pkg_version=-3.5.5.5'
+  .. Note:: The following nodes file is just as a sample. Please use or update the values with your actual deployment. The below nodes file deploys OpenShift version 3.11. To deploy OpenShift version 3.6, use 'openshift_pkg_version=-3.6.173.0.5' or to deploy OpenShift version 3.5, use 'openshift_pkg_version=-3.5.5.5' etc
```

Comment:
Just remove the mentions of older versions, we really should just focus on 3.11 here
```suggestion
  .. Note:: The following nodes file is just as a sample. Please use or update the values with your actual deployment. The below nodes file deploys OpenShift version 3.11. 
```